services/apps/alcs/src/alcs/application-decision/application-component-lot/application-decision-component-lot.entity.ts

```diff
@@ -0,0 +1,73 @@
+import { AutoMap } from '@automapper/classes';
+import { Type } from 'class-transformer';
+import { Column, Entity, ManyToMany, ManyToOne } from 'typeorm';
+import { Base } from '../../../common/entities/base.entity';
+import { ColumnNumericTransformer } from '../../../utils/column-numeric-transform';
+import { ApplicationDecisionConditionToComponentLot } from '../application-condition-to-component-lot/application-decision-condition-to-component-lot.entity';
+import { ApplicationDecisionComponent } from '../application-decision-v2/application-decision/component/application-decision-component.entity';
+
+export class ProposedLot {
+  type: 'Lot' | 'Road Dedication' | null;
+  alrArea?: number | null;
+  size: number | null;
+  planNumbers: string | null;
+}
+
+//Contains the approved subdivision lots
+@Entity()
+export class ApplicationDecisionComponentLot extends Base {
+  constructor(data?: Partial<ApplicationDecisionComponentLot>) {
+    super();
+    if (data) {
+      Object.assign(this, data);
+    }
+  }
+
+  @AutoMap()
+  @Column({ type: 'int' })
+  number: number;
```

Comment:
number is not very descriptive, maybe index or rank?

Reply:
renamed to index